src/main/scala/io/sdkman/changelogs/Neo4jMigrationsMigrations.scala

```diff
@@ -0,0 +1,59 @@
+package io.sdkman.changelogs
+
+import com.github.mongobee.changeset.{ChangeLog, ChangeSet}
+import com.mongodb.client.MongoDatabase
+
+@ChangeLog(order = "072")
+class Neo4jMigrationsMigrations {
+
+  val candidate = "neo4j-migrations"
```

Comment:
We have an informal convention that there are no special characters in the candidate identifier. I'd suggest simply going for `neo4jmigrations`.